src/OpenTelemetry.Api/Metrics/Meter.cs

```diff
@@ -71,6 +55,22 @@ public abstract class Meter
         /// <returns>The measure instance.</returns>
         public abstract Measure<double> CreateDoubleMeasure(string name, bool absolute = true);
 
+        /// <summary>
+        /// Creates Int64 Observer with given name.
+        /// </summary>
+        /// <param name="name">The name of the observer.</param>
+        /// <param name="absolute">indicates if only positive values are expected.</param>
+        /// <returns>The observer instance.</returns>
+        public abstract Observer<long> CreateInt64Observer(string name, bool absolute = true);
```

Comment:
looks like a name clash with the https://docs.microsoft.com/en-us/dotnet/standard/events/how-to-implement-an-observer

Maybe `MetricsObserver`?

Reply:
I understand the concern and agree it's best to disambiguate. However, if we add a prefix, we should apply it consistently and add to all other similar types. eg `MetricsCounter`, `MetricsMeasure`, `MetricsCounterHandle`, etc.

What do you think?

Reply:
A suffix would work: `ObserverMetric`, `ObserverMetricHandle`, etc

Reply:
Applied `Metric` suffix to counter, measure and observer